### ISSUE TITLE
Collaboration with miniwdl

### ISSUE THREAD

**Issue by melibleq** [common-workflow-lab/wdl-cwl-translator#144]
Some issues in this repo could use a chat with the folks working on the miniwdl project (see https://github.com/common-workflow-lab/wdl-cwl-translator/issues?q=is%3Aissue+miniwdl)

Perhaps @kinow could reach out to them?

**Comment by kinow** [comment]
Good idea @melibleq . So far I have only looked at the translator code and at miniwdl API. Might be a good time to start looking at the miniwdl ideas & issues too 👍

**Comment by mr-c** [comment]
Looking is fine, but sending Mike Lin an email (or opening an issue) and letting him know what we are up would be better :-)
